perl_lib/EPrints/Plugin/Screen/EPrint/Move.pm

```diff
@@ -29,7 +29,7 @@ sub new
 { place => "eprint_actions_bar_archive", action => "move_buffer", position => 100, },
 { place => "eprint_actions_bar_archive", action => "move_deletion", position => 100, },
 { place => "eprint_actions_bar_deletion", action => "move_archive", position => 100, },
-{ place => "eprint_review_actions", action => "move_archive", postion => 200, },
+{ place => "eprint_review_actions", action => "move_archive", position => 200, },
```

Comment:
From many years of commercial software development I know that people have lots of problems which they never report or which they report to the wrong addressee with the wrong words. It's also quite common that an old unnoticed problem suddenly occurs very often just in the moment where it was discovered.